src/components/profile/ProfileBasicBusinessCard.tsx

```diff
@@ -189,18 +191,32 @@ const ProfileBasicBusinessCard: React.VFC<ProfileBasicBusinessCardProps> = ({ fo
                 initialValue: defaultValue,
```

Comment:
need to handle display value if form item is select component
<img width="366" alt="image" src="https://user-images.githubusercontent.com/28040359/227755774-aa686e5d-6c1f-48e1-8251-cfed5b486764.png">